src/components/Modals/Onboarding.vue

```diff
@@ -288,8 +287,40 @@ export default {
     this.setCollegesMap();
     this.setMajorsList();
     this.setMinorsList();
+    this.flattenDisplayMajors();
   },
   methods: {
+    flattenDisplayMajors() {
+      const majors = [];
+      this.displayOptions.major.forEach(major => {
+        if (Array.isArray(major.acronym)) {
+          major.acronym.flat(Infinity);
+          for (let i = 0; i < major.acronym.length; i += 1) {
+            const newMajor = {
+              shown: false,
+              stopClose: false,
+              boxBorder: '',
+              arrowColor: '',
+              placeholderColor: '',
```

Comment:
One problem I noticed on onboarding is that after you select a major, close onboarding, and re-open it, any majors selected have gray text instead of black text. Could there be a problem here or somewhere else in your added code that does not set the color properly?